openapi-cli/src/main/java/org/ballerinalang/ballerina/service/OpenApiResourceMapper.java

```diff
@@ -498,7 +498,13 @@ private void createParametersModel(FunctionDefinitionNode resource, Operation op
      */
     private void parseResourceConfigAnnotationAttachment(FunctionDefinitionNode resource, OperationAdaptor operation) {
 
-            operation.setPath(resource.relativeResourcePath().get(0).toString());
+        String path = resource.relativeResourcePath().get(0).toString();
+        if (path.equals(".") || path.equals("/")) {
```

Comment:
Do we need the `path.equals("/")` condition ? since `path` would already be `/`.

Reply:
we can remove that condition, because `post /` type scenarios are invalid in the resource type.